src/objects/DataImportBatch__c.object

```diff
@@ -173,28 +190,123 @@
+        <externalId>false</externalId>
+        <inlineHelpText>Shows the number of records that failed to be successfully processed on the date listed in the Last Processed On field.</inlineHelpText>
+        <label>Records Failed</label>
+        <precision>8</precision>
+        <required>false</required>
+        <scale>0</scale>
+        <trackHistory>false</trackHistory>
+        <trackTrending>false</trackTrending>
+        <type>Number</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Records_Successfully_Processed__c</fullName>
+        <description>Shows the number of records that were successfully processed on the date listed in the Last Processed On field.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>Shows the number of records that were successfully processed on the date listed in the Last Processed On field.</inlineHelpText>
+        <label>Records Successfully Processed</label>
+        <precision>8</precision>
+        <required>false</required>
+        <scale>0</scale>
+        <trackHistory>false</trackHistory>
+        <trackTrending>false</trackTrending>
+        <type>Number</type>
+        <unique>false</unique>
+    </fields>
     <label>NPSP Data Import Batch</label>
+    <listViews>
+        <fullName>All</fullName>
+        <columns>OBJECT_ID</columns>
+        <columns>NAME</columns>
+        <columns>LAST_UPDATE</columns>
+        <columns>Batch_Description__c</columns>
+        <filterScope>Everything</filterScope>
+        <label>All</label>
+        <language>en_US</language>
+    </listViews>
     <nameField>
         <label>Batch Name</label>
+        <trackHistory>false</trackHistory>
         <type>Text</type>
     </nameField>
     <pluralLabel>NPSP Data Import Batches</pluralLabel>
-    <searchLayouts/>
+    <searchLayouts>
+        <lookupDialogsAdditionalFields>Batch_Description__c</lookupDialogsAdditionalFields>
+        <lookupDialogsAdditionalFields>UPDATEDBY_USER</lookupDialogsAdditionalFields>
+        <searchResultsAdditionalFields>Batch_Description__c</searchResultsAdditionalFields>
+        <searchResultsAdditionalFields>UPDATEDBY_USER</searchResultsAdditionalFields>
+    </searchLayouts>
     <sharingModel>ReadWrite</sharingModel>
+    <webLinks>
+        <fullName>Process_Batch</fullName>
+        <availability>online</availability>
+        <description>invoke the Data Importer on all records associated with this Batch</description>
+        <displayType>button</displayType>
+        <encodingKey>UTF-8</encodingKey>
+        <height>600</height>
+        <linkType>url</linkType>
+        <masterLabel>Process Batch</masterLabel>
+        <openType>sidebar</openType>
+        <protected>false</protected>
+        <url>/apex/npsp__BDI_DataImport?batchId={!DataImportBatch__c.Id}&amp;retURL={!URLFOR(&apos;/&apos; + DataImportBatch__c.Id)}</url>
```

Comment:
I'm unable to test this Process Batch button in LEX due to the namespace, we should have this tested in a beta package or during regression.